src/Helpers/FileOutput.php

```diff
@@ -68,9 +67,9 @@ public function write(string $path, array $attributes = []): void
      * @param  string  $path
      * @return string
      */
-    public function assetPath(string $path): string
+    public function assetPath(string $path, bool $dev = false): string
```

Comment:
🚩 Same here. Why are we passing `dev` as an argument here? Isn't that a general setting, why have it as an argument?